tests/config1/src/delay.c

```diff
@@ -0,0 +1,51 @@
+#include "cmsis_os2.h"
+#include <stdio.h>
+#include <stdbool.h>
+#include <unistd.h>
+#include "cmsisPosix_Config.h"
+#include "cmsisPosix_TestInfra.h"
+
+// Thread function prototypes
+void Thread1(void *argument);
+void Thread2(void *argument);
+bool thread1 = true;
+bool thread2 = true;
+void test_start(void) {
+  printf("CMSIS-RTOS2 delay test start\n");
+
+  osKernelInitialize();                 // Initialize CMSIS-RTOS
+
+  osThreadNew(Thread1, NULL, NULL);     // Create Thread1
+  osThreadNew(Thread2, NULL, NULL);     // Create Thread2
+  usleep(100 * 1000);  			// Sleep for 100 ms
+  CP_ASSERT_TRUE(thread1 == true);
```

Comment:
All over the tests here, please use the relevant asserts (most of the time CP_ASSERT_EQ), this will make the prints in case of errors much better